app/controllers/devise_token_auth/sessions_controller.rb

```diff
@@ -48,7 +48,7 @@ def create
     def destroy
       # remove auth instance variables so that after_action does not run
       user = remove_instance_variable(:@resource) if @resource
-      client = @token.client if @token.client
+      client = @token.client
```

Comment:
what does this line really do?

Reply:
just sets the client, there is no need to set it as `client = @token.client if @token.client` the `if` is unnecessary. So instead of doing:
```
client = @token.client if @token.client
```

We could just be doing:

```
client = @token.client
```

I know it is something tiny but in the end, removing the `if` helps with the readability and complexity.

Reply:
yeah but client is an unused variable

Reply:
I think it is being used in line 55:
```
user.tokens.delete(client)
```

Reply:
cool